extras/cronwrapper

```diff
@@ -51,11 +51,11 @@ if $LOGGING; then
     RET=1
   fi
 else
-  "${SCRIPT}" "${CONF}" 2>&1 >${LOGFILE}
+  "${SCRIPT}" "${CONF}" >${LOGFILE} 2>&1
```

Comment:
Because bash parses redirects backwards from what you'd expect. :smile:

`2>&1 >${LOGFILE}` redirects STDOUT to LOGFILE **and then** redirects STDERR to STDOUT (but **not** the log file).

The changed version is what I believe we want, since it redirects both STDOUT and STDERR to a log file. Without it, when DISTRO_INSTALL gets called from within plexupdate, the STDERR output never makes it into the log and instead gets printed to STDOUT which is not expected behavior.